test/models/report_test.rb

```diff
@@ -51,4 +51,13 @@ class ReportTest < ActiveSupport::TestCase
     report.save!
     assert_not_nil Watch.find_by(user: adviser, watchable: report)
   end
+
+  test '#latest?' do
+    assert_not reports(:report31).latest?
+    assert reports(:report32).latest?
+  end
+
+  test '#interval' do
+    assert_equal reports(:report32).interval, 10
```

Comment:
第1引数と第2引数が逆転しているようです。[こちら](https://railsguides.jp/testing.html#%E5%88%A9%E7%94%A8%E5%8F%AF%E8%83%BD%E3%81%AA%E3%82%A2%E3%82%B5%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3)をご覧ください。
```suggestion
    assert_equal 10, reports(:report32).interval
```

Reply:
最初に期待値でした。確認不足ですみません。修正しています:[4eb4256](https://github.com/fjordllc/bootcamp/pull/4170/commits/4eb4256b1689c842f0266bbf5165a2a717f7baa1)